int16 steering
---
bool was_set
